### PR TITLE
Spezifikation Erweiterungen DW

### PR DESCRIPTION
**Pull request by MichaelKohler**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -88,7 +89,20 @@ The returned data is a single entry.
                     "legalFormCode": null,
                     "languageCode": "deu",
                     "blockingCode": null,
-                    "closingDate": null
+                    "closingDate": null,
+                    "activeState": "true (wie bisher aus ODB)",
+                    "gender": "male (wie bisher aus ODB)",
+                    "lettreTitle": "Sehr geehrter Herr Muster (aus Finnova Tabelle KD_STAMM, Feld BRIEFANR)",
```
Comment (hairmare) [inline apiary.apib]: Shouldn't this be letterTitle?

File: apiary.apib [hunk 2]
```diff
@@ -134,7 +149,20 @@ The returned data is a single entry.
...
```
Comment (hairmare) [inline apiary.apib]: same as above...

File: apiary.apib [hunk 3]
```diff
@@ -177,6 +206,19 @@ The returned data is a single entry.
...
```
Comment (hairmare) [inline apiary.apib]: and again...

**Comment by narcoticfresh** [comment]
Just out of curiosity - I don't think this specs here are really relevant anymore in the future as we'll move to swagger-based docs generated from our json specs - so the json specs will be the one and only truth..  or am i missunderstanding something?

**Comment by MichaelKohler** [comment]
I'm using this as drop-off for needed fields for our system (mail to follow to Sabine), so you can move this to whatever doc system you want :)

**Comment by hairmare** [comment]
What @narcoticfresh says means that we'll be removing all the graviton-api-* repos at some point. You might not want to rely on them for posterity.

I suggest we keep this PR open until we worked through this with Sabine and it's implemented at the source.

**Pull request closed without merging**